### PR TITLE
Fix #194470 www.wtae.com

### LINKED ISSUE #194470: www.wtae.com
### Issue URL (Ads)
 [https://www.wtae.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.wtae.com%2F)
Username: `@xglossyntx`

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.4.48 MV2
System version: | Windows 10
Browser: | Firefox
Tracking protection options: | Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Self-destructing third-party cookies (2880), <br>Block trackers
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Russian
Browsing Security: | enabled
Browsing Security statistics: | disabled

### PR DESCRIPTION
**Pull request by xglossyntx**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

Fix #194470

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -18281,6 +18280,9 @@ circleofcricket.com#$##myModal { display: none !important; }
 circleofcricket.com##div.container-full.add:has(> a[href]:first-child)
 ||circleofcricket.com/assets/adv/
 !
+! Hearst Stations Inc
```
Comment (Alex-302) [inline BaseFilter/sections/specific.txt:18283]: In their footer - `Hearst Television Inc`

```suggestion
! Hearst Television Inc
```

Comment (xglossyntx) [reply BaseFilter/sections/specific.txt:18283]: I had not made up my mind about fixing that, as I was thinking about merging their news stations too, such as https://www.chron.com/ and https://www.theintelligencer.com/.

Comment (xglossyntx) [reply BaseFilter/sections/specific.txt:18283]: Should I change it to Hearst Television Inc and leave it as is, or should I merge the news sites into it and change the name to Hearst Communications Inc?

**Comment by xglossyntx** [comment]
@Alex-302 It should be ready to merge.

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
Will rework a bit.

**Comment by xglossyntx** [comment]
Is this rule even needed? ``chron.com##div[data-block-type="ad"]``, since there is a general rule for ``##[data-block-type="ad"]``?

**Comment by xglossyntx** [comment]
There is also this rule ``chron.com,seattlepi.com,sfchronicle.com,timesunion.com##[data-block-type="ad"]``

**Comment by Alex-302** [comment]
https://github.com/AdguardTeam/AdguardFilters/commit/62af0fac2e361bc148410b90d933717f41efad2c

**Pull request merged**